tests/unit/test_advanced.py

```diff
@@ -57,11 +57,16 @@ def initial_feedback(cls):
         return cls.expected_configuration()["initial_feedback"]
 
     def test_get_configuration(self):
-        self.assertEqual(self.expected_configuration(), self.block.get_configuration())
+        try:
+            self.assertEqual(self.expected_configuration(), self.block.get_configuration())
+        except AssertionError:
+            print self.expected_configuration()
+            print self.block.get_configuration()
+            raise
```

Comment:
The problem is that committed code should almost never contain "print", unless you're writing some kind of interactive shell script. So if you leave it in, you should use this syntax:

```python
expected = self.expected_configuration()
actual = self.block.get_configuration()
self.assertEqual(
    expected,
    actual,
    "Unexpected configuration - Expected:\n{}\nObserved:\n{}".format(expected, actual)
)
```

p.s. You should get out of the habit of writing `print "foo"` since it's not python3 compatible :-p